### PR TITLE
Use AdminProperties instead of freeform launcherProps

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes spring-cloud/spring-cloud-dataflow-admin-kubernetes#19

### CODE REVIEW

File: spring-cloud-dataflow-admin-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/DefaultContainerFactory.java [hunk 1]
```diff
@@ -38,6 +39,9 @@
 	@Autowired
 	protected KubernetesModuleDeployerProperties properties;
 
+	@Autowired
+	private AdminProperties adminProperties;
+
```
Comment (markpollack) [inline spring-cloud-dataflow-admin-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/DefaultContainerFactory.java:46]: AdminProperties as well as KubernetesModuleDeployerProperties should change to constructor args.  Will do it on merge.

Comment (markpollack) [inline spring-cloud-dataflow-admin-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/DefaultContainerFactory.java:46]: Current PR fails basic 'contextLoads' style test

```
org.springframework.beans.factory.BeanCreationException: Could not autowire field: private org.springframework.cloud.dataflow.admin.config.AdminProperties org.springframework.cloud.dataflow.module.deployer.kubernetes.DefaultContainerFactory.adminProperties; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [org.springframework.cloud.dataflow.admin.config.AdminProperties]
```

Comment (markpollack) [inline spring-cloud-dataflow-admin-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/DefaultContainerFactory.java:46]: Of note, the same 'contextLoads' test fails for admin-cloudfoundry.  Investigating.

**Comment by ericbottard** [comment]
Addressed review comments

File: spring-cloud-dataflow-admin-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/DefaultContainerFactory.java (hunk 1 above)
Comment (ericbottard) [inline spring-cloud-dataflow-admin-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/DefaultContainerFactory.java:46]: I'm not seeing this (either here or CF). What test is this? I can build this project fine, and run the admin fine

**Comment by markpollack** [comment]
I made the same changes locally and also added a 'contextLoads' test, so will merge what I have already.  CF also has this issue with the added twist that a contextLoads test fails as it tries to make some connection.  I suppose mesos has the same problem.  Didn't check yet, but have we documented these properties in the main dataflow docs?

File: spring-cloud-dataflow-admin-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/DefaultContainerFactory.java (hunk 1 above)
Comment (markpollack) [inline spring-cloud-dataflow-admin-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/DefaultContainerFactory.java:46]: Add a test 

```
@Test
public void contextLoads() {
}
```

**Comment by markpollack** [comment]
Merged in 2bfec294f02afad7fdc449b40f8676e2f191751b

**Pull request closed without merging**